GliaWidgets/SecureConversations/SecureConversations.Availability.swift

```diff
@@ -40,8 +40,16 @@ extension SecureConversations {
             let filteredQueues = fetchedQueues.filter(defaultPredicate)
 
             // Check if matched queues match support `messaging` and
-            // have status other than `closed`
+            // have status other than `closed`.
             guard !filteredQueues.isEmpty else {
+                // In case of "transferred SC" we should treat SC as available.
+                if let engagement = environment.getCurrentEngagement(),
+                   engagement.status == .transferring,
+                   engagement.capabilities?.text == true {
+                    completion(.success(.available(queueIds: [])))
```

Comment:
is it intentional to pass an empty array of `queueIds`?

Reply:
Yes, because in case of transferring there are no queues with messaging. This is also visible from the check above:
```
 guard !filteredQueues.isEmpty else {
```

Reply:
@EgorovEI  @ykyivskyi-gl @rasmustautsglia  Perhaps it make sense to return different result case for clarity?

Reply:
Sure, having dedicated enum case for this case would be much more understandable

Reply:
@ykyivskyi-gl @EgorovEI @rasmustautsglia  Added 'transferred' available state 9cdb97d94df8155f86048825625e93529756582a.